multisig/proposals/opsepolia/vip-006/index.ts

```diff
@@ -0,0 +1,131 @@
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "setMaxTokensDistributionSpeed(address[],uint256[])", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "setMaxLoopsLimit(uint256)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "pauseFundsTransfer()", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "resumeFundsTransfer()", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateAlpha(uint128,uint128)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateMultipliers(address,uint256,uint256)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setStakedAt(address[],uint256[])", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "addMarket(address,address,uint256,uint256)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setLimit(uint256,uint256)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setMaxLoopsLimit(uint256)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "issue(bool,address[])", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "burn(address)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "togglePause()", opsepolia.GUARDIAN],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
+    {
+      target: opsepolia.XVS_VAULT_PROXY,
+      signature: "setPrimeToken(address,address,uint256)",
+      params: [PRIME, opsepolia.XVS, PRIME_POOL_ID],
+    },
+    {
+      target: PRIME,
+      signature: "setLimit(uint256,uint256)",
+      params: [
+        0, // irrevocable
+        0, // revocable
+      ],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "pauseFundsTransfer()",
+      params: [],
+    },
```

Comment:
I would invoke `setPrime` on the Core pool comptroller. To avoid any issues in the future

Reply:
done, https://github.com/VenusProtocol/vips/pull/382/commits/13f3a7a50d5c167ddfc4a04cdd204ef214725332